pypeit/scripts/show_arxiv.py

```diff
@@ -7,7 +7,7 @@ def parse_args(options=None, return_parser=False):
     import argparse
     parser = argparse.ArgumentParser(description='Show an archived arc spectrum',
```

Comment:
It would be good to add in the description that the archived arc spectrum must be in `pypeit/data/arc_lines/reid_arxiv`. It could be not so obvious to some people.